File: src/com_old/rosterWeb/rosterStatistic/MonthlyStatistic.java
package com_old.rosterWeb.rosterStatistic;

public class MonthlyStatistic 
{
	private int aShiftTotal,bxShiftTotal,cShiftTotal,dxShiftTotal,oShiftTotal,monthlyTotal;

	public int getAShiftTotal() {
		return aShiftTotal;
	}

	public void setAShiftTotal(int aShiftTotal) {
		this.aShiftTotal = aShiftTotal;
	}

	public int getBxShiftTotal() {
		return bxShiftTotal;
	}

	public void setBxShiftTotal(int bxShiftTotal) {
		this.bxShiftTotal = bxShiftTotal;
	}

	public int getCShiftTotal() {
		return cShiftTotal;
	}

	public void setCShiftTotal(int cShiftTotal) {
		this.cShiftTotal = cShiftTotal;
	}

	public int getDxShiftTotal() {
		return dxShiftTotal;
	}

	public void setDxShiftTotal(int dxShiftTotal) {
		this.dxShiftTotal = dxShiftTotal;
	}

	public int getOShiftTotal() {
		return oShiftTotal;
	}

	public void setOShiftTotal(int oShiftTotal) {
		this.oShiftTotal = oShiftTotal;
	}

	public int getMonthlyTotal() {
		monthlyTotal=aShiftTotal+bxShiftTotal+cShiftTotal+dxShiftTotal+oShiftTotal;
		return monthlyTotal;
	}	
}

File: server/classes/rosterStatistic/MonthlyStatistic.js
class MonthlyStatistic{
    constructor(){
        this.aShiftTotal=0;
        this.bxShiftTotal=0;
        this.cShiftTotal=0;
        this.dxShiftTotal=0;
        this.oShiftTotal=0;

        this.getMonthlyTotal=()=>{
            return (
                this.aShiftTotal+
                this.bxShiftTotal+
                this.cShiftTotal+
                this.dxShiftTotal+
                this.oShiftTotal
            )
        }
    }
}
module.exports=MonthlyStatistic;